cypress/fixtures/example.json

```diff
@@ -0,0 +1,3 @@
+{
+  "//": "This file will be recreated with sample data everytime you open cypress, so we leave it 'empty'."
```

Comment:
do we need this file?

Reply:
It isn't added anymore every time you open cypress, so it's not necessary anymore.